cluster-scope/overlays/prod/common/kustomization.yaml

```diff
@@ -1,56 +1,57 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-    - ../../../base/config.openshift.io/projects/cluster
```

Comment:
a lot of these seem to be just indent changes, can we put these back to original format to clean up the pr?

Reply:
sure